packages/paste-website/stories/PageTemplates.stories.tsx

```diff
@@ -860,3 +880,505 @@ export const SettingsOverviewExample = (): JSX.Element => {
 SettingsOverviewExample.parameters = {
   padding: false,
 };
+
+export const DefaultWizardExample = (): JSX.Element => {
+  const input1 = useUID();
+  const input2 = useUID();
+  const input3 = useUID();
+  const input4 = useUID();
+  return (
+    <Box display="flex" width="100%" justifyContent="center" paddingX="space10">
+      <Box
+        paddingTop="space130"
+        paddingBottom="space160"
+        display="flex"
+        flexDirection="column"
+        rowGap="space130"
+        width="size70"
+        maxWidth="size80"
+      >
+        <Box display="flex" flexDirection="column">
+          <Box display="flex" rowGap="space30" marginBottom="space130">
+            <Breadcrumb aria-label={useUID()}>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+            </Breadcrumb>
+          </Box>
+          <ProgressSteps>
+            <ProgressStepCurrent as="div">Label</ProgressStepCurrent>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+            <ProgressStepSeparator />
+            <ProgressStepIncomplete as="div">Label</ProgressStepIncomplete>
+          </ProgressSteps>
+          <Box marginTop="space70">
+            <DetailText>Wizard title</DetailText>
+            <Heading as="h4" variant="heading10">
```

Comment:
question: Should these be `as="h1"`?

Reply:
I made them h4s and h5s because the examples will be on the docs pages that have their own heading hierarchy